m4/trilinos.m4

```diff
@@ -197,9 +201,11 @@ AC_DEFUN([CONFIGURE_TRILINOS_10],
        dnl header files whose absence indicates Epetra is not available,
        dnl we just choose one here that libmesh actually includes.
        dnl ------------------------------------------------------
-       AC_CHECK_HEADER([$withtrilinosdir/include/Epetra_CombineMode.h],
```

Comment:
Unless the existence of Epetra_config.h definitely guarantees the other is present... since libmesh directly includes the other.